pipelines/pepatac.py

```diff
@@ -477,6 +495,18 @@ def check_commands(commands, ignore=''):
         return False
 
 
+def _add_resources(args, res):
+    # TODO: how to name this? consider env vars?
+    rgc = RGC(select_genome_config(res.get("genome_config")))
+    from attmap import EchoAttMap
+    translations = EchoAttMap({"TSS_file": "tss_annotation"})
```

Comment:
I like that you're using the attmap functionality here but this will be pretty hard to read for someone who hasn't seen this before...do we really need an `EchoAttMap` here for this?

Instead let's just match the actual asset names... so, say, rename `TSS_file` in pepatac to `tss_annotation`?

Reply:
Done